db/schema.rb

```diff
@@ -10,19 +10,19 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.1].define(version: 2023_12_21_025246) do
+ActiveRecord::Schema[7.1].define(version: 2024_01_20_011344) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "pgcrypto"
   enable_extension "plpgsql"
 
   create_enum :invitation_status, [
     "pending",
-    "sent",
     "accepted",
     "rejected",
     "expired",
     "ignored",
     "revoked",
+    "sent",
```

Comment:
This looks like an accident, may want to `git checkout -p main db/schema.rb` and get rid of the stuff that's not related to the change you're making

Reply:
Perhaps one of these days I'll memorize the patch options 🤣

`[y,n,q,a,d,j,J,g,/,s,e,?]` 😳😳😳😳😳😳😳😳